src/components/Helper.js

```diff
@@ -0,0 +1,51 @@
+/* eslint no-eval: 0 */
```

Comment:
This helper file is fine. But it doesnt belong in the components directory. It should have a directory /helper and this file should be quiz.js. As it has helper functions for quiz

Reply:
Done